kubejs/server_scripts/gregtech/circuits.js

```diff
@@ -145,7 +172,7 @@ ServerEvents.recipes(event => {
     // WIP: Dimensional Circuits
     event.recipes.gtceu.circuit_assembler('dimensional_circuit_board')
         .itemInputs(
-            '32x kubejs:matter_circuit_board',
```

Comment:
qb says matter circuit boards will be used to craft remaining boards as well; this would change that

Reply:
Fixed in latest commit